### ISSUE TITLE
How to troubleshoot the cause of warning

### ISSUE THREAD

**Issue by wnark** [gravitl/netmaker#613]
![image](https://user-images.githubusercontent.com/24355466/149775044-625882bc-8ab4-4754-b120-dc99e5904683.png)
Hello, as shown in the picture, there is a warning in the web interface, but there is no error pop-up window. How can I screen it?
It can ping other nodes, I don't know why it gives the warning
my version is 0.92

**Comment by wnark** [comment]
![image](https://user-images.githubusercontent.com/24355466/149776268-2c91bd0b-67c3-4183-9963-56f83d38f034.png)
On the client side, it can be seen that the communication takes place in about 1 minute, but there is no successful update information on the web.

**Comment by afeiszli** [comment]
@wnark did you try just refreshing the UI or opening in incognito? It may be a simple caching error.

The "Warning" only means that the node has not contacted the server in more than 5 minutes. It should go into "Error" status if it does not contact the server in 30+ minutes. If it is stuck in "warning" there is likely a separate issue.

Typically, nodes go into Warning and then Error if the daemon is not running correctly (it runs a local checkin every 15 seconds). I would check the local daemon logs to see if that indicates an issue.

**Comment by wnark** [comment]
Hello, my problem here may be solved.
My abnormal server is using the Alibaba Cloud Shanghai server. On the 17th, I tried to use this server as an access point and found that even if I opened all ports, my wireguard client still could not connect. This server, I guess is that the udp port of Alibaba Cloud Shanghai area is blocked by ddos. I don't know if the check-in service is using the udp protocol to update the check-in?
Now I see the server is fine on the control node and I can also access this server through the wireguard client.
Therefore, on the 17th, it is normal for the control panel to prompt a warning, because udp is not working.
